scala-unit-tests/src/test/scala/org/eclipse/collections/impl/set/mutable/MultiReaderUnifiedSetScalaTest.scala

```diff
@@ -19,6 +19,13 @@ class MultiReaderUnifiedSetScalaTest extends MultiReaderUnifiedSetTestTrait
 {
     val classUnderTest = MultiReaderUnifiedSet.newSetWith(1, 2, 3)
 
+    @Test
+    override def equals_safe(): Unit =
```

Comment:
I don't think this override is really needed, in my PR it was needed because I did some other changes related to `equals` and `hashCode` that made `equals` non-blocking for the `null` case (the overriden test calls `equals(null)`).